.github/workflows/auto-assign-to-project.yml

```
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request_target:
+    types: [opened, labeled]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.AUTO_ASSIGN_WORKFLOW_TOKEN }}
+
+jobs:
+  assign-to-project:
+    runs-on: ubuntu-latest
+    name: Assign to Project
+    steps:
+    - name: Assign pull requests to "Unprioritised"
+      uses: srggrs/assign-one-project-github-action@1.2.0
```

Could you update the version to [1.2.1](https://github.com/srggrs/assign-one-project-github-action/releases/tag/1.2.1)?

```suggestion
      uses: srggrs/assign-one-project-github-action@1.2.1
```

---

Looks like yesterday was not a good day for me to do anything good 😅 
Sorry for this, should be fixed now!